draft/context.jsonld

```diff
@@ -1,7 +1,51 @@
 {
   "@context": {
     "@version": 1.1,
+    "ocfl": "https://ocfl.io/draft/spec#",
+    "dcterms": "http://purl.org/dc/terms/",
     "id": "@id",
-    "type": "@type"
+    "type": "@type",
+
+    "Object": "ocfl:Object",
+    "Version": "ocfl:Version",
+
+    "digestAlgorithm": {
+      "@id": "ocfl:digestAlgorithm"
+    },
+    "head": {
+      "@id": "ocfl:head"
+    },
+
+    "manifest": {
+      "@id": "ocfl:manifest"
+    },
+    "versions": {
+      "@id": "ocfl:versions",
+      "@container": "@list",
+      "@type": "ocfl:Version"
+    },
+    "fixity": {
+      "@id": "ocfl:fixity"
+    },
+
+    "version": {
+      "@id": "ocfl:version"
+    },
+    "created": {
+      "@id": "dcterms:created",
+      "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+    },
+    "message": {
```

Comment:
`message`: Is this intended as a "comment" on OCFL:Objects?

Reply:
I think it's a comment on a version, like a commit message.